src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,145 @@
+
+        if (orders[0].equals("bye")) {
+            response = ui.bye();
+        } else if (orders[0].equals("list")) {
+            response = ui.list(tasks.getTasks());
+        } else if (orders[0].equals("unmark")) {
+            try {
+                int number = parser.parseUnmark(input, tasks.getSize());
+                tasks.unmark(number);
+                response = ui.unmark(tasks.getTask(number));
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("mark")) {
+            try {
+                int number = parser.parseMark(input, tasks.getSize());
+                tasks.mark(number);
+                response = ui.mark(tasks.getTask(number));
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("todo")) { 
+            try {
+                Task task = parser.parseTodo(input);
+                tasks.addTask(task);
+                response = ui.addedMessage(task) + ui.totalTask(tasks.getSize());
+                storage.save(tasks.getTasks());
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("deadline")) { 
+            try {
+                Task task = parser.parseDeadline(input);
+                tasks.addTask(task);
+                response = ui.addedMessage(task) + ui.totalTask(tasks.getSize());
+                storage.save(tasks.getTasks());
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("event")) { 
+            try {
+                Task task = parser.parseEvent(input);
+                tasks.addTask(task);
+                response = ui.addedMessage(task) + ui.totalTask(tasks.getSize());
+                storage.save(tasks.getTasks());
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("delete")) {
+            try {
+                int number = parser.parseDelete(input, tasks.getSize());
+                response = ui.deletedMessage(tasks.getTask(number));
+                tasks.deleteTask(number);
+                response += ui.totalTask(tasks.getSize());
+                storage.save(tasks.getTasks());
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else if (orders[0].equals("find")) {
+            try {
+                String word = parser.parseFind(input);
+                TaskList newtasks = new TaskList(tasks.findTasks(word));
+                response = ui.findList(newtasks.getTasks());
+            } catch (DukeException e) {
+                response = e.getMessage();
+            }
+        } else {
+            response = ui.dontUnderstand();
+        }
+        return response;
+    }
```

Comment:
Should this be extracted out? For example you can extract one method for Parser class and one method for Ui class instead of combining them in the Duke class. Or create a new method only for extracting the numbers in a command.